Include/internal/pycore_interp.h

```diff
@@ -188,8 +189,7 @@ struct _is {
     struct _dtoa_state dtoa;
     struct _py_func_state func_state;
 
-    struct _Py_tuple_state tuple;
```

Comment:
@colesbury 
dict_state was not removed at the previous commit, this was why I didn't catch the weird behavior. :(
And also _Py_tuple_state should be removed before.